main/modes/games/pango/pango.c

```diff
@@ -269,6 +269,9 @@ void pangoEnterMode(void)
     pango->tilemap.entityManager    = &(pango->entityManager);
     pango->tilemap.tileSpawnEnabled = true;
 
+    pa_loadMapFromFile(&(pango->tilemap), "preset.bin");
```

Comment:
@JVeg199X do you know why this is in my PR? I merged from `main` first and I didn't add this myself

Reply:
Eh, whatever this was, I undid it.